.sonarcloud.properties

```diff
@@ -1,5 +1,6 @@
 sonar.projectKey=keptn_lifecycle-toolkit
 sonar.projectName=lifecycle-toolkit
+sonar.exclusions=test/kuttl/**/*.yaml
```

Comment:
why are you ignoring this folder for ALL sonarcloud checks? is there an option to reduce this to a smaller scope to have some sonar check still enabled for the folder?

Reply:
ok i checked, it seems there's no specific option for security hotspots which were the thing this was added i think